alembic/ddl/postgresql.py

```diff
@@ -848,3 +880,39 @@ def _render_potential_column(
             autogen_context,
             wrap_in_element=isinstance(value, (TextClause, FunctionElement)),
         )
+
+
+def add_column(
+    compiler: DDLCompiler,
+    column: Column[Any],
+    *,
+    if_not_exists: Optional[bool] = None,
+    **kw,
+) -> str:
+    text = "ADD COLUMN "
+    if if_not_exists:
+        text += "IF NOT EXISTS "
+
```

Comment:
why not put this in [base.py](http://base.py/)? if I have a migration that is adding / dropping columns that may or may not exist already, can't it be assumed I'm using a database that supports this syntax? would I want if_exists / if_not_exists to be silently ignored if I passed it?

even if we dont want it to render we should likely keep this in [base.py](http://base.py/) anyway and then control for when we actually render IF [NOT] EXISTS

Reply:
I pushed the move to base.py as a distinct commit for you to validate, I'll squash changes prior to merging if you confirm it's ok for you.